manifests/params.pp

```diff
@@ -11,7 +11,8 @@
   $config_defaults       = {}
   $config_dir            = '/etc/consul'
   $config_hash           = {}
-  $config_mode           = '0660'
+  $config_mode           = '0664'
```

Comment:
Can you leave it at 660 to continue to allow potentially sensitive configs unreadable by random users of the system or hacked nobodys or whatever?